react/components/ProductImages/index.js

```diff
@@ -120,6 +121,18 @@ ProductImages.getSchema = ({ zoomProps: { zoomType } = {} }) => {
               default: 0.8,
             },
           }),
+          ...(zoomType === 'in-page' && {
+            desktopTrigger: {
```

Comment:
Did you check if this props should have an `isLayout` option?

Reply:
They should, but this option is currently not working on the cms